lib/ui/widgets/office_settings.dart

```diff
@@ -123,6 +168,9 @@ class OfficeSettings extends StatelessWidget {
                             lngController.text,
                             authRangeController.text,
                             dateIpServiceUrlController.text,
+                            starTimeController.text,
```

Comment:
save to firebase as proper .toIso8601String()
https://api.dart.dev/stable/2.7.1/dart-core/DateTime/toIso8601String.html